Cargo.toml

```diff
@@ -17,7 +17,7 @@ license = "MIT"
 name = "rogcat"
 readme = "README.md"
 repository = "https://github.com/flxo/rogcat"
-version = "0.4.6"
+version = "0.4.9-pre"
```

Comment:
This is a huge jump. Think this should be 0.4.7-pre and additionally not part of this pr. I'll fix it.

Reply:
I guess I got a bit to excited with those version numbers, I changed it to 0.4.7-pre hope that's fine for your